### PR TITLE
Add spec

### PR DESCRIPTION
**Pull request by uu59**

### CODE REVIEW

File: spec/jira_api_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -- coding: utf-8
```
Comment (yoshihara) [inline spec/jira_api_spec.rb]: :thought_balloon: If we will support ruby > 2.0 only,  this line doesn't needed...?

Comment (uu59) [inline spec/jira_api_spec.rb]: :persevere:

Comment (nahi) [inline spec/jira_api_spec.rb]: Could you send issue? https://github.com/embulk/embulk/blob/master/embulk-core/src/main/java/org/embulk/jruby/JRubyScriptingModule.java#L66

Should be 2.0!

Comment (uu59) [inline spec/jira_api_spec.rb]: oh.. but that magic comment was unneeded, so remove it is no problem for now.
Thank you for your information!

**Comment by yoshihara** [comment]
LGTM

**Pull request merged**